src/feature/NodeGraphFeature.ts

```diff
@@ -0,0 +1,183 @@
+'use strict';
```

Comment:
Git nit: could this have been a file rename instead of a delete? Lost history...etc

Reply:
Rebased and fixed.